### DISCUSSION TITLE
Bun

### DISCUSSION THREAD

**Discussion by calumk** [talentlessguy/tinyhttp#405] [General]
Have you performed any tests with Bun? 

It seems to work from my basic testing - but dont know how performant it is

**Comment by calumk** [comment]
I guess because it relies on "node:http" - it wont be as quick as it could be?

**Reply by v1rtl** [reply]
I haven't tried Bun on tinyhttp, since it runs the same as Node.js there's no additional support required.

**Reply by v1rtl** [reply]
Some of the packages of tinyhttp are now powered with Bun, works without issue

I'll also add an example to tinyhttp repo soon
